diskquota--1.0.3--2.0.sql

```diff
@@ -0,0 +1,284 @@
+-- TODO check if worker should not refresh, current lib should be diskquota-2.0.so
+
+-- table part
+ALTER TABLE diskquota.quota_config ADD COLUMN segratio float4 DEFAULT 0;
+
+CREATE TABLE diskquota.target (
+	rowId serial,
+	quotatype int, -- REFERENCES disquota.quota_config.quotatype,
+	primaryOid oid,
+	tablespaceOid oid, -- REFERENCES pg_tablespace.oid,
+	PRIMARY KEY (primaryOid, tablespaceOid, quotatype)
+);
+-- TODO ALTER TABLE diskquota.target SET DEPENDS ON EXTENSION diskquota;
+
+ALTER TABLE diskquota.table_size ADD COLUMN segid smallint DEFAULT -1; -- segid = coordinator means table size in cluster level
+ALTER TABLE diskquota.table_size DROP CONSTRAINT table_size_pkey;
+ALTER TABLE diskquota.table_size ADD PRIMARY KEY (tableid, segid);
+ALTER TABLE diskquota.table_size SET WITH (REORGANIZE=true) DISTRIBUTED BY (tableid, segid);
+
+-- TODO SELECT pg_catalog.pg_extension_config_dump('diskquota.target', '');
+-- TODO SELECT gp_segment_id, pg_catalog.pg_extension_config_dump('diskquota.target', '') FROM gp_dist_random('gp_id');
```

Comment:
@RekGRpth, can you explain why they can't mark newly created table as needed to dump?

Reply:
I have no any idea why updating does not mark new table `diskquota.target` as needed to dump, while clean installing does it.

Reply:
comment was removed